slave.mk

```diff
@@ -535,7 +535,19 @@ $(addprefix $(TARGET_PATH)/, $(DOCKER_IMAGES)) : $(TARGET_PATH)/%.gz : .platform
 		--build-arg frr_user_uid=$(FRR_USER_UID) \
 		--build-arg frr_user_gid=$(FRR_USER_GID) \
 		--label Tag=$(SONIC_GET_VERSION) \
-		-t $* $($*.gz_PATH) $(LOG)
+		-t $* $($*.gz_PATH) $(LOG) ||
+	echo "Re-trying with no squashing" &&
+	docker build --no-cache \
+		--build-arg http_proxy=$(HTTP_PROXY) \
+		--build-arg https_proxf=$(HTTPS_PROXY) \
+		--build-arg user=$(USER) \
+		--build-arg uid=$(UID) \
+		--build-arg guid=$(GUID) \
+		--build-arg docker_container_name=$($*.gz_CONTAINER_NAME) \
+		--build-arg frr_user_uid=$(FRR_USER_UID) \
+		--build-arg frr_user_gid=$(FRR_USER_GID) \
+		--label Tag=$(SONIC_GET_VERSION) \
+		-t $* $($*.gz_PATH) $(LOG) 
```

Comment:
why this change?

Reply:
This is needed if we want to keep the namespace container, since the docker build fails when it tries to squash the container

Reply:
so, it is tight to namespace container, let me resolve this in the conversation above.